### ISSUE TITLE
Support of tvOS for library and design toolbox

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#1109]
We support with v0.21.0 iOS / iPadOS, macOS and visionOS.
tvOS support should be studied for both package and app.

**Comment by B3nz01d** [comment]
@pylapp how can this be tested for me as the app does not launch on tvos?
